app/pages/Auction/RepairBid.js

```diff
@@ -57,13 +57,13 @@ export default class RepairBid extends Component {
 
   processValue = async (val) => {
     const value = val.match(/[0-9]*\.?[0-9]{0,6}/g)[0];
-    const dollarydoos = parseFloat(value) * consensus.COIN;
-    if (dollarydoos > consensus.MAX_MONEY)
-      return;
     this.setState({value: value});
-    if (val === "")
+    const parsed = parseFloat(value);
+
+    if (val === "" || Number.isNaN(parsed) || parsed * consensus.COIN > consensus.MAX_MONEY)
```

Comment:
I think `setState` needs to come after this check, so if the user tries to type `999999999999999`, it will just freeze once they get to MAX_MONEY, etc.

Reply:
Nothing happens if you type a value too high.
The user doesn't know that we try to validate on each key pressed. In this case, we just don't try to validate.
It's harmless and transparent.

Reply:
ok fair enough!